### ISSUE TITLE
Aria-describedby attributes appear without matching description in the DOM

### ISSUE THREAD

**Issue by mjulstein** [Altinn/app-frontend-react#23]
## Describe the bug
Tested on a text input.
The aria-describedby attribute is added to all input fields regardless if there actually is an element on the page that has the given id.

## To Reproduce
1. Open an app with inputs.
2. Inspect the input input element in the browser.
3. Notice that there is an aria-describedby attribute on all [text] inputs, but not necessarily an element with the id inside the DOM.

## Expected behavior
Either the element that is referred exists in the DOM with the correct id or the attribute is not set.

## Screenshots
![image](https://user-images.githubusercontent.com/6736558/139084225-1aba472e-4bef-4d6c-923c-24048b64c6e9.png)

## Additional info
> This doesn't seem to be a big problem as I cannot find anything stating that it is illegal.
> Getting rid of these attributes would make the DOM cleaner and reduce the overall size of the client-side app, as the id's often end up being quite long.

**Comment by lorang92** [comment]
Fix merged, unit test added for all cases + manual verification.
Closing issue as done.
